chromadb/utils/embedding_functions.py

```diff
@@ -637,6 +639,64 @@ def __call__(self, input: Union[Documents, Images]) -> Embeddings:
         return embeddings
 
 
+class RoboflowEmbeddingFunction(EmbeddingFunction[Union[Documents, Images]]):
+    def __init__(
+        self, api_key: str, api_url = "https://infer.roboflow.com"
+    ) -> None:
+        self._api_url = api_url
+        self._api_key = api_key
```

Comment:
since in the documentation we suggest `export`ing the API key, could we read it in here, then error if it's still none?

Reply:
Could we also add docstring here, for the parameters?